docs/docs/en/architecture/design.md

```diff
@@ -46,27 +52,26 @@
      Server provides monitoring services based on netty.
   
      #### The Service Mainly Includes:
-  
-     - **Fetch TaskThread** is mainly responsible for continuously getting tasks from the **Task Queue**, and calling **TaskScheduleThread** corresponding executor according to different task types.
+
+    - **WorkerManagerThread** is mainly responsible for the submission of the task queue, continuously receives tasks from the task queue, and submits them to the thread pool for processing;
+
+    - **TaskExecuteThread** is mainly responsible for the process of task execution, and the actual processing of tasks according to different task types;
+
+    - **RetryReportTaskStatusThread** is mainly responsible for regularly polling to report the status of the task to the Master until the Master replies to the status ack to avoid the loss of the task status;
```

Comment:
to report the status of the task
->
 to report the task status

Reply:
done